apps/ui/components/single-search/SearchForm.tsx

```diff
@@ -86,15 +87,23 @@ const Filters = styled.div<{ $areFiltersVisible: boolean }>`
 
   @media (min-width: ${breakpoints.m}) {
     margin-top: var(--spacing-s);
-    grid-template-columns: 1fr 1fr;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-columns: repeat(3, auto);
+    > div {
+      grid-column: span 1;
+    }
   }
 
   @media (min-width: ${breakpoints.l}) {
-    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-columns: repeat(3, 1fr);
   }
 `;
 
-const StyledSelect = styled(Select<OptionType>)`
+const Filters = styled.div<{ children: ReactNode[] }>`
```

Comment:
Where is the `children` used?

Reply:
Nowhere anymore. It's a relic from the former Filters `grid`, which was constructed dynamically based on `children.length`. Now it's removed. 👍